db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/TableDescriptor.java

```diff
@@ -399,11 +399,11 @@ public void setPurgeDeletedRows(boolean purgeDeletedRows) {
         this.purgeDeletedRows = purgeDeletedRows;
     }
 
-    public Long getMinRetainedVersions() {
+    public Long minRetainedPeriod() {
```

Comment:
should it be in Java style getMinRetainedPeriod?

Reply:
Done